netlogo-gui/src/main/app/infotab/InfoFormatter.scala

```diff
@@ -44,12 +44,12 @@ object InfoFormatter {
                          MaxParsingTimeMillis)
       .markdownToHtml(str)
 
-  def wrapHtml(body: HTML, fontSize: Int = defaultFontSize): HTML = {
-    "<html><head>"+styleSheet(fontSize)+"</head><body>"+body+"</body></html>"
-  }
+  def wrapHtml(body: HTML, fontSize: Int = defaultFontSize): HTML =
+    "<html><head>"+styleSheet(fontSize)+"</head><body>"+
+      body.replaceAll("<li>", "<li>&bull; ")+
+      "</body></html>"
 
-  def apply(content: String, fontSize: Int = defaultFontSize, attachModelDir: String => String = identity) = {
```

Comment:
This parameter doesn't seem to be used, so I removed it. Have I missed something?